src/main/java/com/darkender/plugins/containerpassthrough/ContainerPassthrough.java

```diff
@@ -33,14 +33,27 @@
     @Override
     public void onEnable()
     {
+
+        saveDefaultConfig();
+
         passthroughEntities = new HashSet<>();
-        passthroughEntities.add(EntityType.PAINTING);
-        passthroughEntities.add(EntityType.ITEM_FRAME);
+
+        if (getConfig().getBoolean("through-painting")) {
+            passthroughEntities.add(EntityType.PAINTING);
+        }
+
+        if (getConfig().getBoolean("through-itemframe")) {
+            passthroughEntities.add(EntityType.ITEM_FRAME);
+        }
+
+        if (getConfig().getBoolean("through-painting")) {
```

Comment:
looks like this is repeated twice?

Reply:
I am dumb lol sorry went too fast

should be fixed